File: Math/ReverseOfGivenNumber/ReverseGivenNumber.java
public class ReverseGivenNumber {
    public static void main(String[] args) {
        // logic
        // int n
        // rev = 0
        // lastDigit = n % 10
        // rev = (rev * 10) + lastDigit
        int n = 10899;
        int rev = 0;

        while (n > 0) {
            int lastDigit = n % 10;
            rev = (rev * 10) + lastDigit;
            n = n / 10;
        }

        System.out.println(rev);
    }
}

File: Math/ReverseOfGivenNumber/reverse_given_number.rs
fn main() {
    // logic
    // int n
    // rev = 0
    // lastDigit = n % 10
    // rev = (rev * 10) + lastDigit
    let mut n: u32 = 10899;
    let mut rev: u32 = 0;

    while n > 0 {
        let last_digit = n % 10;
        rev = (rev * 10) + last_digit;
        n = n / 10;
    }

    println!("Reverse of the number: {}", rev);
}
